manager/manager/typeresolver.go

```diff
@@ -122,14 +132,39 @@ func (tr *typeResolver) ResolveTypes(config *common.Configuration, imports []*co
 				fmt.Errorf("Number of imports exceeds maximum of %d", tr.maxUrls))
 		}
 
-		url := toFetch[0]
-		template, err := performHTTPGet(tr.getter, url, false)
-		if err != nil {
-			return nil, resolverError(config, err)
+		templates := []string{}
+		url := toFetch[0].urls[0]
```

Comment:
Is this URL special? Is it documented anywhere? might be worth breaking the fetch unit into template and imports if this guy is a special one.